test/restricted-post-test.js

```diff
@@ -290,9 +290,16 @@ suite.addBatch({
                             cb = this.callback,
                             url = doc.object.likes.url;
 
-                        httputil.getJSON(url, cred, function(err, likes, response) {
-                            cb(err, likes);
-                        });
+                        Step(
+                            function(){
+                                setTimeout(this,100);
```

Comment:
The httputil.getJSON wants to get likes that aren't there yet.
Not the most elegant solution. It would be better to wait for a callback that the likes are set. But I couldn't find such a solution.

#### Open for suggestions